sdk/include/opentelemetry/sdk/trace/samplers/parent_or_else.h

```diff
@@ -9,21 +9,7 @@ namespace sdk
 namespace trace
 {
 namespace trace_api = opentelemetry::trace;
-
-/**
- * A placeholder class that stores the states of a span.
- * Will be replaced by the real SpanContext class once it is implemented.
- */
-class Sampler::SpanContext
-{
-public:
-  inline explicit SpanContext(bool is_recording, bool sampled_flag)
-      : is_recording(is_recording), sampled_flag(sampled_flag)
-  {}
-
-  bool is_recording;
-  bool sampled_flag;
-};
+namespace trace_sdk = opentelemetry::sdk::trace;
```

Comment:
This should be `trace_api` and `opentelemetry::trace`.

Reply:
Updated